### PR TITLE
Adds onboarding introduction page

### PR DESCRIPTION
**Pull request by ConorOkus**
This page is the introductory landing page for the onboarding chapter. It outlines the sections in this chapter.

Preview - https://deploy-preview-134--sad-borg-390916.netlify.app/guide/onboarding/introduction/

Closes https://github.com/BitcoinDesign/Guide/issues/129

### CODE REVIEW

File: guide/onboarding/introduction.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
+permalink: /guide/onboarding/introduction/
```
Comment (GBKS) [inline guide/onboarding/introduction.md:7]: Will you add a preview image later?

File: guide/onboarding/introduction.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 
+
+On the other hand, more than four billion people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.
+
+Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements). But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added. 
+
+Remember: onboarding should not be a crutch for bad design. Avoid over indexing on trying to explain too much within onboarding, and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great app experience.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:29]:
```suggestion
Remember: Onboarding should not be a crutch for bad design. Avoid trying to explain too much and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great user experience.
```

Simplified a little bit. The short paragraph used the term "onboarding" three times which felt repetitive.

File: guide/onboarding/introduction.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 
+
+On the other hand, more than four billion people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.
+
+Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements). But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:27]: IIRC, you copy also focuses primarily on mobile wallets for consumers. If so, is that something worth pointing out?

File: guide/onboarding/introduction.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:23]:
```suggestion
Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard, as they might assume that the patterns and habits from prior experience will apply. Convincing them to use a different financial system might be better in the product. 
```

Not totally clear what you mean by `might be better in the product.`. As opposed to what?

File: guide/onboarding/introduction.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+Editor's notes
+
+A brief introduction and summary of all pages in this section. The idea is that readers
+scan this page to get an overview of the section and then decide which topics to dive into.
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:21]:
```suggestion
Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind. 
```

This is just a suggestion. Something doesn't flow quite right with the original text. One thing is that it's not only the Bitcoin experience, but the product experience overall. Also wanted to reduce the duplicate use of the word "audience" in the second sentence. Also wanted to clarify what onboarding is right away. Just a few ideas for you to consider.

File: guide/onboarding/introduction.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+-->
+
+# Onboarding
+
+Effective onboarding can make or break someone’s first bitcoin experience. Therefore, it is important to identify your target audience when building wallet-products and design specifically with that audience in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes them tricky to onboard. Users might assume that the patterns and habits from prior experience will apply, so convincing them to use a different financial system might be better in the product. 
+
+On the other hand, more than four billion people have no exposure to traditional financial institutions. This, along with designing for younger people whose ideas about money aren’t fully formed, represents a very different challenge.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:25]: Backing this up with research would be great. No worries if there is no good reference.

File: guide/onboarding/introduction.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements). But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added. 
+
+Remember: onboarding should not be a crutch for bad design. Avoid over indexing on trying to explain too much within onboarding, and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great app experience.
+
+---
+
+**Principles (coming soon)**
+
+Onboarding experiences can look very different depending on your target audience, however there are some things that should be consistent across bitcoin products.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:35]:
```suggestion
Onboarding experiences can look very different depending on your target audience, however there are some things that should be consistent across Bitcoin products.
```

**Comment by GBKS** [comment]
Good start.

Small thing, what are your thoughts on separating lines between pages like in the [Getting Started](http://localhost:4000/guide/getting-started/introduction/) intro. Private Key Management doesn't have them, but we should standardize.

**Comment by danielnordh** [comment]
> Small thing, what are your thoughts on separating lines between pages like in the [Getting Started](http://localhost:4000/guide/getting-started/introduction/) intro. Private Key Management doesn't have them, but we should standardize.

I only did lines before and after the page descriptions as each paragraph is so short I didn't think it necessary.
Happy to change though, no big deal and agree it should be standardized.

File: guide/onboarding/introduction.md (hunk 6 above)
Comment (danielnordh) [reply guide/onboarding/introduction.md:25]: I would strike or rethink this phrase: "This makes them tricky to onboard."
Them, is basically every new user to bitcoin and this reads as them being unusual.

File: guide/onboarding/introduction.md (hunk 1 above)
Comment (ConorOkus) [reply guide/onboarding/introduction.md:7]: Yeah I've updated this now, was waiting on some illustrations but I'll put something in for now and change later.

File: guide/onboarding/introduction.md (hunk 4 above)
Comment (ConorOkus) [reply guide/onboarding/introduction.md:36]: Trying to get across the idea that the experience of using a bitcoin product should be self-evident that it is better than the traditional financial system.

File: guide/onboarding/introduction.md (hunk 5 above)
Comment (ConorOkus) [reply guide/onboarding/introduction.md:34]: I prefer this actually, the use of wallet-product early on can be a bit confusing

File: guide/onboarding/introduction.md (hunk 6 above)
Comment (ConorOkus) [reply guide/onboarding/introduction.md:38]: The number seems to be closed to 2 Billion as well according to the World Bank - https://globalfindex.worldbank.org/sites/globalfindex/files/chapters/2017%20Findex%20full%20report_chapter2.pdf

Although I feel like I have definitely seen larger numbers. 

@danielnordh think I agree, will update 🚧

Comment (ConorOkus) [reply guide/onboarding/introduction.md]: Updated, let me know your thoughts

**Comment by ConorOkus** [comment]
> Good start.
> 
> Small thing, what are your thoughts on separating lines between pages like in the [Getting Started](http://localhost:4000/guide/getting-started/introduction/) intro. Private Key Management doesn't have them, but we should standardize.

Yeah I like it, will update 🚧

File: guide/onboarding/introduction.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Onboarding
+
+Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind. 
+
+Some new users bring years of traditional financial experience to bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to swtich to a different financial system will be a lot easier.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:36]:
```suggestion
Some new users bring years of traditional financial experience to Bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to swtich to a different financial system will be a lot easier. 
```

**Review by pavlenex — COMMENTED** [review]
Concept ACK.

File: guide/onboarding/introduction.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Onboarding
+
+Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind. 
+
+Some new users bring years of traditional financial experience to Bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to swtich to a different financial system will be a lot easier.
```
Comment (pavlenex) [inline guide/onboarding/introduction.md:36]:
```suggestion
Some new users bring years of traditional financial experience to Bitcoin. They have bank accounts, debit cards, and ingrained ideas about what money is. This makes onboarding tricky, as the user might assume that the patterns and habits from prior experience will apply. If you design a user experience that is superior to the traditional system, convincing users to switch to a different financial system will be a lot easier. 
```

File: guide/onboarding/introduction.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+Onboarding, for this guide, refers to anything that helps users set up the product or learn its interface (such as dedicated flows and UI elements) and initially focuses on mobile applications. But onboarding isn’t just for first-time users. Experienced users also need to be onboarded, especially when new features are added. 
+
+Remember: Onboarding should not be a crutch for bad design. Avoid trying to explain too much and then shooting users into a poorly designed app. Rather, think about onboarding flows as a smooth on-ramp into a great user experience.
+
+---
+
+**Principles (coming soon)**
+
+Onboarding experiences can look very different depending on your target audience, however there are some things that should be consistent across Bitcoin products.
```
Comment (pavlenex) [inline guide/onboarding/introduction.md:48]: Maybe this sounds better?
```suggestion
Onboarding experiences can look very different depending on your target audience, however, some things should be consistent across Bitcoin products.
```

File: guide/onboarding/introduction.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
+---
+layout: guide
+title: Onboarding
+description: An overview of some of the concepts to consider when building onboarding experiences.
+nav_order: 2
+has_children: true
```
Comment (GBKS) [inline guide/onboarding/introduction.md:6]: You can remove `has_children` if there are no sub-pages. With that property enabled, you can expand the section, even when there are no sub pages.

**Review by pavlenex — APPROVED** [review]

**Review by GBKS — APPROVED** [review]
tACK

tACK

**Pull request merged**